addons/escoria-core/game/esc_inputs_manager.gd

```diff
@@ -380,6 +394,18 @@ func _on_mouse_left_double_clicked_item(
 	item: ESCItem,
 	event: InputEvent
 ) -> void:
+	if item as ESCPlayer and not (item as ESCPlayer).selectable:
```

Comment:
Same as above:
Shouldn't this block happen after test if input_mode == INPUT_ALL: that happens below?